nixops/backends/__init__.py

```diff
@@ -237,7 +257,9 @@ def send_keys(self):
               )
             )
             os.remove(tmp)
-        self.run_command("touch /run/keys/done")
+        self.run_command("mkdir -m 0750 -p /run/keys && "
+                         "chown root:keys  /run/keys && "
```

Comment:
@ip1981, that's because all key might have an overridden `destinationFolder`, but we need a well-known location for the `done` flag -- and that`s `/run/keys`..

Just think about it as if the snippet at lines 214-215 was moved down, here.  Makes it simpler to understand the change, really.